hack/helm_vars/redis-cluster/values.yaml.gotmpl

```diff
@@ -1,8 +1,3 @@
-global:
-  storageClass: {{ .Values.storageClass }}
```

Comment:
this is used for both rabbitmq and redis. removing this here will just lead to helm templating errors, you need to also remove the usage of this; as well as disable persistence in rabbitmq.

Reply:
nevermind, I'm wrong.